overlay/lower/usr/sbin/sysupgrade

```diff
@@ -156,7 +156,7 @@ Where:
   -f      full upgrade with a binary from GitHub
   -p      partial upgrade with a binary from GitHub
   -b      upgrade only bootloader
-  -d      release date (the latest available, if omitted)
+  -d      release date as YYYY-MM-DD (the latest available, if omitted)
```

Comment:
the script can handle any date format supported by `date`. it will convert to the required format automatically.